### DISCUSSION TITLE
How can we enable the previous and next button?

### DISCUSSION THREAD

**Discussion by xihajun** [squidfunk/mkdocs-material#6995] [Q&A] [answered]
https://github.com/squidfunk/mkdocs-material/blob/f27b93ece3c423537873e0bc5de55b3c3381792f/docs/setup/changing-the-logo-and-icons.md?plain=1#L80-L85

**Comment by alexvoss** [comment, marked as answer]
This is part of the footer, so the info is in the [setup guide for the footer](https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-footer/#navigation).

**Reply by xihajun** [reply]
thanks a lot!
